cli/eac.js

```diff
@@ -369,37 +275,44 @@ const main = async (_) => {
       console.log("  error: must be running a localnode on the Ropsten or Kovan networks")
       process.exit(1)
     }
-    if (!await eac.Util.checkForUnlockedAccount()) process.exit(1)
+
+    if (!program.wallet || !program.password) throw new Error('Use a wallet');
+
+    const wallet = new Wallet(web3);
+    wallet.decrypt(JSON.parse(fs.readFileSync(program.wallet[0])), program.password);
```

Comment:
I would detangle this line for better readability.

Reply:
Can add this in to a later PR.

Reply:
2018: // Temporary: Detangle this in the next PR
2122: // Don't touch, magic